object-cache.php

```diff
@@ -1247,10 +1247,11 @@ public function build_client_parameters( $redis_server ) {
 			// Attempt to automatically load Pantheon's Redis config from the env.
 			if ( isset( $_SERVER['CACHE_HOST'] ) ) {
 				$redis_server = [
-					'host' => wp_strip_all_tags( $_SERVER['CACHE_HOST'] ),
-					'port' => isset( $_SERVER['CACHE_PORT'] ) ? wp_strip_all_tags( $_SERVER['CACHE_PORT'] ) : $port,
-					'auth' => isset( $_SERVER['CACHE_PASSWORD'] ) ? wp_strip_all_tags( $_SERVER['CACHE_PASSWORD'] ) : null,
-					'database' => isset( $_SERVER['CACHE_DB'] ) ? wp_strip_all_tags( $_SERVER['CACHE_DB'] ) : $database,
+					'host' => strip_tags( $_SERVER['CACHE_HOST'] ),
```

Comment:
Our linting is yelling about replacing `wp_strip_all_tags` with `strip_tags` as `strip_tags` does not strip CSS and JS in between the script and style tags. Is there a reason you've made the change?

Reply:
Yes, see the other reported issues linked in this PR that are being fixed.